### DISCUSSION TITLE
Namespaces for custom metadata fields

### DISCUSSION THREAD

**Discussion by monotasker** [MESH-Research/knowledge-commons-repository#74] [General]
FYI, Invenio's custom metadata fields can be namespaced. So I'm creating the custom fields we need under to different namespaces: "kcr" and "hclegacy". The "kcr" namespace will hold all of the metadata that we're going to actively use, but that doesn't fit into Invenio's default schema. The "hclegacy" namespace will hold fields for legacy data that we're keeping (and that we might later use for things like making community associations) but that isn't part of the metadata schema we want to use moving forward.

**Comment by bjr70** [comment]
This makes perfect sense to me.
